browser-interface/packages/shared/dao/sagas.ts

```diff
@@ -216,6 +262,15 @@ function* cacheCatalystRealm() {
   const dao: string = yield select((state) => state.dao)
   const realmAdapter: IRealmAdapter = yield call(waitForRealm)
 
+  if (
+    isOnboarding &&
+    realmAdapter.about.configurations?.realmName !==
+      getFeatureFlagVariantValue(store.getState(), 'new_tutorial_variant')
```

Comment:
* where does `isOnboarding` comes from?
* can you unwind this if into `if (two && consts)`?

Reply:
isOnboarding is a bool that determines we are doing the onboarding. I needed some way to detect that we were inOboarding when a realmChanged happen, so as to set the onboarding as complete. I could as well have started another async function that checks that, but not completely sure on how to do it.
That would have saved me the isOnboarding var